.github/workflows/create_release.yml

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Create a new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: "Define the release version (e.g., v4.7.0)"
+        required: true
+        default: ""
+      development-version:
```

Comment:
You can use this to determine the next development version from the release version.
```
          # Assumes semantic versioning and increments the minor version. Adjust the awk command as needed for different versioning schemes.
          NEXT_VERSION=$(echo ${VERSION} | awk -F. '{print $1"."$2+1".0-SNAPSHOT"}')
```

Reply:
I like current version, you set what is the release version and what is the development version. Above assumes that we always increment the same digit all the time. But I will remove comment because that is incorrect

Reply:
I'm not going to argue about such a thing, but in that case you need to make sure that the development version will end with `-SNAPSHOT`